.factory/automation.yml

```diff
@@ -34,21 +34,29 @@ build:
     build:
       image: vaticle-ubuntu-22.04
       command: |
+        bazel run @vaticle_dependencies//tool/bazelinstall:remote_cache_setup.sh
         bazel build //...
         bazel run @vaticle_dependencies//tool/checkstyle:test-coverage
         bazel test $(bazel query 'kind(checkstyle_test, //...)') --test_output=errors
     build-dependency:
       image: vaticle-ubuntu-22.04
       command: |
+        bazel run @vaticle_dependencies//tool/bazelinstall:remote_cache_setup.sh
```

Comment:
this one shouldn't use the bazel cache since it's not run with bazel

Reply:
As discussed, we're adding this.